### ISSUE TITLE
Using alpha version of Storybook

### ISSUE THREAD

**Issue by AndreasBBS** [react-theming/storybook-addon#60]
Sorry to bother, I'm using @storybook/*@6.5.0-alpha.36 because of some emotion issues I was having. So far everything is working and I can use it locally but when I run install on my pipelines I get the following error.

```
$ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @react-theming/storybook-addon@1.1.5
npm ERR! Found: @storybook/react@6.5.0-alpha.36
npm ERR! node_modules/@storybook/react
npm ERR!   dev @storybook/react@"^6.5.0-alpha.36" from the root project
npm ERR!   peer @storybook/react@"*" from @storybook/addon-devkit@1.4.2
npm ERR!   node_modules/@storybook/addon-devkit
npm ERR!     @storybook/addon-devkit@"^1.4.2" from @react-theming/storybook-addon@1.1.5
npm ERR!     node_modules/@react-theming/storybook-addon
npm ERR!       dev @react-theming/storybook-addon@"^1.1.5" from the root project
npm ERR!   1 more (@storybook/addon-docs)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @storybook/react@"^5.0.0 || ^6.0.0" from @react-theming/storybook-addon@1.1.5
npm ERR! node_modules/@react-theming/storybook-addon
npm ERR!   dev @react-theming/storybook-addon@"^1.1.5" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @storybook/react@6.4.22
npm ERR! node_modules/@storybook/react
npm ERR!   peer @storybook/react@"^5.0.0 || ^6.0.0" from @react-theming/storybook-addon@1.1.5
npm ERR!   node_modules/@react-theming/storybook-addon
npm ERR!     dev @react-theming/storybook-addon@"^1.1.5" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /root/.npm/eresolve-report.txt for a full report.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-05-03T23_00_20_503Z-debug-0.log
Cleaning up project directory and file based variables 00:01
ERROR: Job failed: command terminated with exit code 1
```

I don't really know how to address this. Any tip?

**Comment by usulpro** [comment]
Hey @AndreasBBS 
It looks like this behavior is caused by the latest version of NPM you use. Please check doesn't this helps https://stackoverflow.com/a/64987777/5721315
If not we can soften the peerDeps requirements in the next release

BTW does this Storybook alpha solves the issues with Emotion ThemeProvider? We also experiencing problems with this

**Comment by AndreasBBS** [comment]
The legacy-peers flag didn't do it for me but the --force one solved it but it doesn't look so stable. I've had other times where I used the --force flag and when running Storybook it failed.
I think it would be nice if the peer dependencies allowed for the beta and alpha releases, not sure how easy it's to do.

That alpha version did solve the emotion ThemeProvider issue for me. I moved it to the alpha in February so the beta might already have those changes at the moment, I went to check and the beta is at 6.5.0 but not the current yet.
